### PR TITLE
[IOTDB-519] Apache IoTDB integration with Prometheus

### PR DESCRIPTION
**Pull request by yanhongwangg**
support prometheus write data to iotdb
https://cwiki.apache.org/confluence/display/IOTDB/Iotdb+integrated+Prometheus

### CODE REVIEW

**Comment by neuyilan** [comment]
Please create or link an issue to describe the intent of this PR, So that people can understand your work better.
I remember there one issue and one design document link to this PR. please link here.

**Comment by yanhongwangg** [comment]
> Please create or link an issue to describe the intent of this PR, So that people can understand your work better.
> I remember there one issue and one design document link to this PR. please link here.
i have modified

**Comment by neuyilan** [comment]
> > Please create or link an issue to describe the intent of this PR, So that people can understand your work better.
> > I remember there one issue and one design document link to this PR. please link here.
> > i have modified

Where is the design document?

File: docs/UserGuide/Ecosystem Integration/Prometheus.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+## Requirement
```
Comment (neuyilan) [inline docs/UserGuide/Ecosystem Integration/Prometheus.md:22]: It's better to indict that the iotdb's version's which iotdb-prometheus-connector support.

```
Support IoTDB version : V0.12 and above
Support prometheus version : XXX
```

File: docs/zh/UserGuide/Ecosystem Integration/Prometheus.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+## ����
+* GoLang 1.13+
```
Comment (neuyilan) [inline docs/zh/UserGuide/Ecosystem Integration/Prometheus.md:23]: the same as above.

File: prometheus_connector/pom.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>iotdb-parent</artifactId>
+        <groupId>org.apache.iotdb</groupId>
+        <version>0.12.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>prometheus_connector</artifactId>
```
Comment (neuyilan) [inline prometheus_connector/pom.xml:29]:
```suggestion
    <artifactId>prometheus-connector</artifactId>
```

**Review by neuyilan — CHANGES_REQUESTED** [review]

File: docs/UserGuide/Ecosystem Integration/Prometheus.md (hunk 1 above)
Comment (yanhongwangg) [reply docs/UserGuide/Ecosystem Integration/Prometheus.md:22]: copy that

File: docs/zh/UserGuide/Ecosystem Integration/Prometheus.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+-->
+
+## ����
+* ֧�ֵ�IoTDB�汾 : V0.12+
+* ֧�ֵ�Prometheus�汾 : V2.23+
+* ֧�ֵ�Go�汾 : V1.13+
+
+## ��IoTDB
+
+�μ�https://iotdb.apache.org/zh/UserGuide/Master/Get%20Started/QuickStart.html
```
Comment (HTHou) [inline docs/zh/UserGuide/Ecosystem Integration/Prometheus.md:29]: Please be careful with the Chinese doc encoding...
See this https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-1052?filter=allissues
Screen Shot 2021-01-13 at 3 33 17 PM

Comment (yanhongwangg) [reply docs/zh/UserGuide/Ecosystem Integration/Prometheus.md]: It's over, I'll pay attention next time

**Pull request closed without merging**